suites/quincy/rbd/tier-2_rbd_mirror_scale_osd.yaml

```diff
@@ -0,0 +1,224 @@
+#===============================================================================================
```

Comment:
These are new test suite and required to add in pipeline also for 5.3 and 6.0. Please add it.

Reply:
Done.

Reply:
looks good